### PR TITLE
Added fastq sync code

### PR DESCRIPTION
**Pull request by ffinfo**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.5")
+addSbtPlugin("com.lucidchart" % "sbt-scalafmt" % "1.10")
```
Comment (rhpvorderman) [inline project/plugins.sbt]: Lucidchart also has a neo scalafmt version and there is also a sbtplugin created by the maintainer of scalafmt http://scalameta.org/scalafmt/#sbt . Have these been considered?

File: project/build.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+sbt.version = 0.13.16
```
Comment (rhpvorderman) [inline project/build.properties]: Newline is missing.

File: src/main/scala/nl/biopet/tools/fastqsync/ArgsParser.scala [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import java.io.File
+
+import nl.biopet.utils.tool.AbstractOptParser
+
+class ArgsParser(cmdName: String) extends AbstractOptParser[Args](cmdName) {
+
+  head(s"""
+          |$cmdName - Sync paired-end FASTQ files.
+          |
+        |This tool works with gzipped or non-gzipped FASTQ files. The output
```
Comment (rhpvorderman) [inline src/main/scala/nl/biopet/tools/fastqsync/ArgsParser.scala]: Indentation seems a bit of for this line

File: project/plugins.sbt (hunk 1 above)
Comment (ffinfo) [reply project/plugins.sbt]: Yes I did look into that but this one seems to work more cleanly inside a complete sbt workflow

File: project/build.properties (hunk 2 above)
Comment (ffinfo) [reply project/build.properties]: added this in all build.properties files in all tools/utils

**Pull request merged**